scripts/bert/pretraining_utils.py

```diff
@@ -126,13 +127,16 @@ def __iter__(self):
 
     return DummyIter(data_batch)
 
-def save_params(step_num, model, trainer, ckpt_dir):
+def save_params(step_num, model, ckpt_dir):
```

Comment:
this name can cause confusion as we advertise that people should use save_parameters.